samples/server/src/main/java/io/oasp/gastronomy/restaurant/general/configuration/ServletInitializer.java

```diff
@@ -1,12 +1,12 @@
 package io.oasp.gastronomy.restaurant.general.configuration;
 
-import io.oasp.gastronomy.restaurant.SpringBootApp;
-
 import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.builder.SpringApplicationBuilder;
 import org.springframework.boot.context.web.SpringBootServletInitializer;
 import org.springframework.context.annotation.Configuration;
 
+import io.oasp.gastronomy.restaurant.SpringBootApp;

```

Comment:
This is just formatting. Should not have been committed. However, since the file has been moved it had to be committed anyways.